bin/test-state-check.sh

```
@@ -0,0 +1,119 @@
+#
+# Not a Unity suite and not counted in test/native-suite-count - the same arrangement as
+# bin/test-config-check.sh, and for the same reason: what it asserts is the behaviour of a process,
+# not of a linkable function.
+#
+# Usage: ./bin/test-state-check.sh   (exit 0 = all fixtures behaved)
+
+set -uo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+ROOT_DIR="$(cd "$SCRIPT_DIR/.." && pwd)"
+cd "$ROOT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)